external/fv3fit/fv3fit/reservoir/domain.py

```diff
@@ -8,3 +14,147 @@ def slice_along_axis(arr: np.ndarray, inds: slice, axis: int = 0):
     sl = [slice(None)] * arr.ndim
     sl[axis] = inds
     return arr[tuple(sl)]
+
+
+class RankDivider:
```

Comment:
I'm guessing this gets used in your fv3fit training code, which is not in this PR.

Reply:
Correct.